openlibrary/plugins/upstream/account.py

```diff
@@ -769,6 +776,51 @@ def GET(self, username, key='loans'):
         raise web.seeother(user.key)
 
 
+class public_my_books_json(delegate.page):
+    encoding = "json"
+    path = "/people/([^/]+)/books/([a-zA-Z_-]+)"
+
+    def GET(self, username, key='want-to-read'):
+        i = web.input(page=1, limit=5000)
```

Comment:
The 5000 limit makes me a little nervous? Let's try it on testing...

Reply:
The existing /account/export interface has a limit of 10000, so I figured I was being conservative... :)